css/properties/visibility.json

```
@@ -62,8 +62,8 @@
               "chrome": {
                 "version_added": "1",
                 "notes": [
-                  "Chrome treats `visibility: collapse` like `hidden`, leaving a white gap.",
-                  "Chrome supports the `collapse` value only on [`&lt;tr&gt;`](https://developer.mozilla.org/docs/Web/HTML/Reference/Elements/tr), [`&lt;thead&gt;`](https://developer.mozilla.org/docs/Web/HTML/Reference/Elements/thead), [`&lt;tbody&gt;`](https://developer.mozilla.org/docs/Web/HTML/Reference/Elements/tbody), and [`&lt;tfoot&gt;`](https://developer.mozilla.org/docs/Web/HTML/Reference/Elements/tfoot), but not on [`&lt;col&gt;`](https://developer.mozilla.org/docs/Web/HTML/Reference/Elements/col) and [`&lt;colgroup&gt;`](https://developer.mozilla.org/docs/Web/HTML/Reference/Elements/colgroup) elements."
+                  "Before Chrome 62, Chrome treats `visibility: collapse` like `hidden`, leaving a white gap.",
+                  "Before Chrome 92, Chrome does not support the `collapse` value on [`&lt;col&gt;`](https://developer.mozilla.org/docs/Web/HTML/Reference/Elements/col) and [`&lt;colgroup&gt;`](https://developer.mozilla.org/docs/Web/HTML/Reference/Elements/colgroup) elements."
```

Let's continue to be explicit about what's supported (but remove those verbose MDN links):

```suggestion
                  "Before Chrome 92, `visibility: collapse` supports `<tr>`,`<thead>`, `<tbody>`, and `<tfoot>` elements, but it has no effect on `<col>` and `<colgroup>` elements."
```